config/ceres_bundle_adjuster.conf

```diff
@@ -78,10 +78,10 @@ ceres:optimize_aspect_ratio = false
 ceres:optimize_dist_k1 = true
 
 # Include radial lens distortion parameter k2 in bundle adjustment.
-ceres:optimize_dist_k2 = false
+ceres:optimize_dist_k2 = true
```

Comment:
I don't think these settings change anything because above we have
```
ceres:lens_distortion_type = NO_DISTORTION
```
These "optimize" setting seem redundant, but the switch above tells you with model to use and these settings tell you whether to keep those parameters fixed in the optimization.  We should probably have k1 turned off as well for consistency.  It also doesn't do anything.

FYI, it's not a good idea to turn on distortion right now because the depth map estimation and depth map fusion do not use the distortion parameters (last I knew).  So if distortion is minor, it's better to have the best fit model without distortion so that dense operation are consistent.  This is something we need to fix before we can process really distorted data like GoPro.